waku/v2/protocol/lightpush/waku_lightpush.go

```diff
@@ -66,23 +66,23 @@ func (wakuLp *WakuLightPush) IsClientOnly() bool {
 
 func (wakuLP *WakuLightPush) onRequest(s network.Stream) {
 	defer s.Close()
-
+	logger := wakuLP.log.With(logging.HostID("peer", s.Conn().RemotePeer()))
```

Comment:
I've used this patter in most of the `onRequest` methods, just attach the peer ID to the logger right at the top and then you get it consistently in all the following log entries.